### PR TITLE
Add more properties to check their order

### PR DESCRIPTION
**Pull request by adamkudrna**
Add more properties to check their order

Properties added (sorted alphabetically):

- `accent-color`
- `backdrop-filter`
- `backface-visibility`
- `background-blend-mode`
- `border-spacing`
- `box-decoration-break`
- `caret-color`
- `color-scheme`, `color-interpolation`
- `column-fill`, `column-span`
- `contain` (including all long-hand variants)
- `container` (including all long-hand variants)
- `counter-set`
- `direction`
- `empty-cells`
- `font-size-adjust`, `font-variant-*`, `font-feature-settings`, `font-stretch`, `font-synthesis` (including all long-hand variants), `font-kerning`, `font-optical-sizing`, `font-palette`
- `image-orientation`, `image-rendering`
- `inset` (including all long-hand variants),
- `justify-items`
- `list-style-image`
- `mask` (including all long-hand variants), `mask-border` (including all long-hand variants), `mask-type`
- `mix-blend-mode`
- `outline-*`
- `overflow-anchor`
- `overscroll-behavior` (including all long-hand variants).
- `paint-order`,
- `perspective`, `perspective-origin`
- `place-content`, `place-items`, `place-self`
- `quotes`
- `scrollbar-*`
- `table-layout`
- `text-align-last`, `text-indent`, `text-decoration-skip-ink`, `text-underline-*`, `text-rendering`, `text-wrap` (including all long-hand variants)
- `touch-action`,
- `transform-box`, `transform-style`, individual transforms: `translate`, `rotate`, `scale`,
- `transition-behavior`
- `white-space-collapse`
- `word-spacing`

The key is roughly like this: all properties from MDN that are standard, widely supported,
non-experimental, not specific for non-western languages, and not designed for paged media.

https://developer.mozilla.org/en-US/docs/Web/CSS

### CODE REVIEW

**Comment by dlouhak** [comment]
Thanks a lot! 💪🏻  I am only missing CSS property `mask`. Could you please add into the `effects and transforms` block?

File: order.js [hunk 1]
```diff
@@ -70,20 +88,27 @@ module.exports = {
...
       'column-rule',
       'column-rule-width',
       'column-rule-style',
       'column-rule-color',
       'gap',
       'column-gap',
       'row-gap',
+      'place-content',
+      'place-items',
+      'place-self',
```
Comment (crishpeen) [inline order.js:103]: bit confusing as align-self/content/items is in a different order.

File: order.js [hunk 2]
```diff
@@ -70,20 +88,27 @@ module.exports = {
       'grid-row-gap',
       'columns',
       'column-count',
+      'column-fill',
       'column-width',
+      'column-span',
```
Comment (crishpeen) [inline order.js:93]: why not above width?

File: order.js [hunk 3]
```diff
@@ -32,16 +32,34 @@ module.exports = {
...
+      'inset-block-start',
+      'inset-block-end',
       'top',
       'right',
       'bottom',
       'left',
+      'contain',
+      'contain-intrinsic-block-size',
+      'contain-intrinsic-height',
+      'contain-intrinsic-inline-size',
```
Comment (crishpeen) [inline order.js:49]: if we already sort
`inline`
`block`

should these be too in the same order?

File: order.js [hunk 4]
```diff
@@ -32,16 +32,34 @@ module.exports = {
...
+      'contain-intrinsic-size',
+      'contain-intrinsic-width',
```
Comment (crishpeen) [inline order.js:51]: `width` is before `height` why not here too?

File: order.js (hunk 2 above)
Comment (crishpeen) [reply order.js:93]: what is the rule here?

File: order.js [hunk 5]
```diff
@@ -231,6 +292,7 @@ module.exports = {
       'border-bottom-right-radius',
       'border-bottom-left-radius',
       'border-collapse',
+      'border-spacing',
```
Comment (crishpeen) [inline order.js:295]: we forgot this one? :O

File: order.js [hunk 6]
```diff
@@ -242,22 +304,59 @@ module.exports = {
       'background-origin',
       'background-clip',
       'background-color',
+      'background-blend-mode',
       'stroke',
       'fill',
+      'paint-order',
```
Comment (crishpeen) [inline order.js:310]: I would probably expect to first set the order and then set the values. This way I would have to do it the other way around. What do you think?

File: order.js (hunk 2 above)
Comment (adamkudrna) [reply order.js:93]: > what is the rule here?

https://developer.mozilla.org/en-US/docs/Web/CSS/column-span

**Pull request merged**